explore/src/main/scala/explore/targets/TargetSelectionPopup.scala

```diff
@@ -301,16 +287,7 @@ object TargetSelectionPopup:
                     )
                   )
                 )
-              }.toTagMod
-            )
-          )(
-            ^.autoComplete.off,
-            ^.onSubmit ==> (e =>
```

Comment:
It is no longer a form that can be submitted, but that shouldn't matter since search is done when the text changes. I'm not sure what actually submitted this, anyway, since there was no `submit` button.

Reply:
This allowed search to be triggered by pressing <ENTER> (text change search waits a few milliseconds after you stop typing).

Reply:
Ah. Do you think that is important? I think I could use a standard form and do that.

Reply:
I personally like it, but I don't think it's absolutely necessary.

Reply:
Such that pressing enter does a submit?

Reply:
I just put a form around the input and it worked.